app/routes/docs/schema-browser/SchemaBrowserElements.lib.tsx

```diff
@@ -0,0 +1,139 @@
+import { Link } from '@remix-run/react'
+import { Icon, Table } from '@trussworks/react-uswds'
+import { useState } from 'react'
+
+export type ReferencedSchema = {
+  $ref: string
+  type: string
+  schema?: Schema
+}
+
+export type SchemaProperty = {
+  description: string
+  type?: string
+  enum?: string[]
+  $ref?: string
+}
+
+export type Schema = {
+  $id?: string
+  $schema?: string
+  type: string
+  title?: string
+  description?: string
+  properties?: { [key: string]: SchemaProperty }
+  $defs?: { [key: string]: SchemaProperty }
+  anyOf?: ReferencedSchema[]
+  allOf?: ReferencedSchema[]
+  oneOf?: ReferencedSchema[]
+  required?: string[]
+}
+
+function ReferencedElementRow({ item }: { item: ReferencedSchema }) {
+  const [showHiddenRow, toggleHiddenRow] = useState(false)
+  const locallyDefined = item.$ref?.startsWith('#')
+
+  return (
+    <>
+      <tr onClick={() => toggleHiddenRow(!showHiddenRow)}>
+        {locallyDefined ? (
+          <>
+            <td colSpan={2}>
+              {item.$ref && item.$ref.split('/').slice(-1)[0]}
+            </td>
+            <td>See below</td>
+          </>
+        ) : (
+          <>
+            <td colSpan={2}>
+              {showHiddenRow ? <Icon.ExpandLess /> : <Icon.ExpandMore />}
+              <Link to={formatLinkString(item.$ref ?? '')}>
+                {item.$ref && item.$ref.split('/').slice(-1)[0]}
+              </Link>
+            </td>
+            <td>
+              {item.$ref && item.$ref.split('/').slice(-2)[0]} schema object{' '}
+              <small>(click to {showHiddenRow ? 'collapse' : 'expand'})</small>
+            </td>
+          </>
+        )}
+      </tr>
+      {!locallyDefined && item.schema && showHiddenRow && (
+        <SchemaPropertiesTableBody schema={item.schema} indent />
+      )}
+    </>
+  )
+}
+
+export function ReferencedElementTable({
+  items,
+}: {
+  items: ReferencedSchema[]
+}) {
+  return (
+    <Table fullWidth stackedStyle="headers">
+      <thead>
+        <tr>
+          <th colSpan={2}>Name</th>
+          <th>Description</th>
+        </tr>
+      </thead>
+      <tbody>
+        {items.map((item) => (
+          <ReferencedElementRow item={item} key={item.$ref} />
+        ))}
+      </tbody>
+    </Table>
+  )
+}
+
+export function SchemaPropertiesTableBody({
+  schema,
+  indent,
+}: {
+  schema: Schema
+  indent?: boolean
+}) {
+  return (
+    <>
+      {schema.properties &&
+        Object.keys(schema.properties).map((itemKey) => (
+          <tr key={itemKey}>
+            <th scope="row" className={indent ? 'indent' : ''}>
```

Comment:
USWDS already provides a utility class for this:

```suggestion
            <th scope="row" className={indent ? 'text-indent-2' : ''}>
```

Reply:
Thats handy, I'll commit this and check it out locally to make sure it looks right